Tests/DependencyInjection/AutowiringExtensionConfigTreeTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Skrz\Bundle\AutowiringBundle\Tests\DependencyInjection;
+
+use PHPUnit_Framework_TestCase;
+use Skrz\Bundle\AutowiringBundle\DependencyInjection\SkrzAutowiringExtension;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\Exception\InvalidTypeException;
+use Symfony\Component\Config\Definition\Processor;
+
+class SkrzAutowiringExtensionConfigTreeTest extends PHPUnit_Framework_TestCase
+{
```

Comment:
This is contradict to: https://github.com/skrz/autowiring-bundle/pull/2#discussion_r35131569, isn't it?